inst-support-eng/panda-pto
# Add agent_sync rake task

closes #79 

#### Test Plan
1. Deploy to test environment
1. Ensure all env vars are set: `MCO_EMAIL` , `FLEET_EMAIL` , `GOOGLE_APPLICATION_CREDENTIALS` , `AGENT_MASTER_SHEET`
1. In the agents google sheet, update an existing user, and create a new user
1. Remote into the test env and run `rake sync_agnets`
1. Verify users from step 3 were updated/created.
